cypress/page-objects/conjuPage/ConjBoxSuiteEle.js

```diff
@@ -0,0 +1,15 @@
+class ConjBoxSuiteEle{
```

Comment:
and in the name of the class `ConjBoxSuiteEle{`
we can skip using `Ele` because it is by default only elements/selectors

also please use the full name of the page it can be useful for you in the future

like so `ConjugationPage`